src/components/auth/landing.js

```diff
@@ -0,0 +1,321 @@
+                Login
+              </a>
+            </li>
+            <li>
+              <a class="btn btn-primary" href="/signup.html">
+                Sign up
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        <section class="header__section container">
+          <div class="header-about p-4">
+            <div class="header__about__text">
+              <h3>Tap into the Power of Community Learning.</h3>
+              <p class="">
+                Community learning is a great way to learn, especially ones with
+                dedicated mentors. Code Clan Code Clan Nigeria is the perfect
+                place for you.
+              </p>
+              <button class="btn btn-dark header__cta rounded-0">
+                Get Started
+              </button>
+            </div>
+
+            <div class="header__image">
+              <img src={building_blocks} class="img-fluid" alt="Header" />
+            </div>
+          </div>
+          <div class="clan_cards">
+            <div class="description__cards">
+              <div class="description__card">
+                <img
+                  src="./img/real_time_collaboration.png"
+                  class="img-fluid"
+                  alt=""
+                  srcset=""
+                />
+                <p>Get assigned to a clan</p>
+              </div>
+              <div class="description__card">
+                <img
+                  src="./img/dev_productivity.png"
+                  class="img-fluid"
+                  alt=""
+                  srcset=""
+                />
+                <p>Start your Journey</p>
+              </div>
+              <div class="description__card">
+                <img
+                  src="./img/assigned_mentor.png"
+                  class="img-fluid"
+                  alt=""
+                  srcset=""
+                />
+                <p>Get Assigned to A mentor</p>
+              </div>
+              <div class="description__card">
+                <img
+                  src="./img/porfolio_wireframe.png"
+                  class="img-fluid"
+                  alt=""
+                  srcset=""
+                />
+                <p>Work on projects/ Build a solid Portfolio</p>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <div class="description__section py-5">
+          <div class="container">
+            <div class="description__goals">
+              <img src="./img/goal_target.png" class="img-fluid" alt="" />
+              <div class="goals__texts">
+                <h3>Tap into the Power of Community Learning.</h3>
+                <p class="my-3">
+                  Community learning is a great way to learn, especially ones
+                  with dedicated mentors. Code Clan Code Clan Nigeria is the
+                  perfect place for you.
+                </p>
+                <a href="#">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5641179)